src/Plugin.php

```diff
@@ -234,6 +239,35 @@ public static function xmlrpc_methods($methods) {
     return $methods;
   }
 
+  /**
+   * @implements wp_head
+   */
+  public static function wp_head() {
+    $data = [
+      'ajaxurl' => admin_url('admin-ajax.php'),
+      'consent_version' => CORE_STANDARDS_COOKIE_CONSENT_VERSION,
+    ];
+    echo '<script>var core_standards = ' . json_encode($data) . ';</script>';
+  }
+
+  /**
+   * Logs cookie consent message.
+   */
+  public static function log_cookie_consent() {
+    $consent = $_POST['consent'] ?? [];
+    $data = [
+      'consent' => $consent,
+      'timestamp' => current_time('timestamp'),
```

Comment:
This should be the first element in the logged data.

Why are we logging machine values that cannot be read?  I'd suggest to change the value to `date_i18n('r')` or `date_i18n('c')` instead.